pwa/src/styling/design-tokens/component-overrides.css

```diff
@@ -6,9 +6,6 @@
     --web-app-secondary-top-nav-font-size
```

Comment:
I see you've removed the hover design tokens here, I'm assuming you did that because you added default values to the component package. Even though that _does_ work (right now) as intended, this is not the best way to do this.

**Component package**

- Includes generic components, with generic styling;
- This generic styling is completely customisable with design-tokens (`--conduction-....` tokens);
- **Important**: this initial, generic, styling is only _initial_ and not actually intended to be the final styling within an implementation (e.g. Open Catalogi).

**Implementation (e.g. Open Catalogi):**

- Whenever including a component from the component package (e.g. the `PrimaryTopNav`);
- We add all available tokens to `theme.css.example` (this is the example theme we can just copy and paste when making a new theme);
- We add new tokens from `theme.css.example` to all theme implementations (in this case: Utrecht and Rotterdam);
- We set the new values within `utrecht.css` and `rotterdam.css` to the values relative to that theme (they might be identical, but they don't _have_ to be, making it customisable);
- We add all new tokens within `component-overrides` to the relative `--conduction` token, to **override** the initial, generic, styling from the components.

The difference between what's stated above and what we've now done in this pull request: we're explicitly telling each theme how it should render the components, instead of implicitly just having the generic styling from the component package set the styling.